diff --git a/internal/shared/logutil/log.go b/internal/shared/logutil/log.go
index 258b340..3b83a94 100644
--- a/internal/shared/logutil/log.go
+++ b/internal/shared/logutil/log.go
@@ -2,6 +2,9 @@ package logutil

 type Func func(format string, args ...interface{})

+// add line 1
+// add line 2
+
 type Log interface {
        Fatalf(format string, args ...interface{})
		Errorf(format string, args ...interface{})
